binding/flutter/README.md

```diff
@@ -0,0 +1,122 @@
+# Cheetah Binding for Flutter
+
+## Cheetah Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Cheetah is an on-device speech-to-text engine. Cheetah is:
+
+- Private; All voice processing runs locally.
+- Accurate [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Compact and Computationally-Efficient [[2]](https://github.com/Picovoice/speech-to-text-benchmark#rtf)
+- Cross-Platform:
+    - Linux (x86_64)
+    - macOS (x86_64, arm64)
+    - Windows (x86_64)
+    - Android
+    - iOS
+    - Raspberry Pi (4, 3)
+    - NVIDIA Jetson Nano
+
+## Compatibility
+
+This binding is for running Cheetah on **Flutter 1.20.0+** on the following platforms:
+
+- Android 4.4+ (API 19+)
```

Comment:
5.0?

Reply:
We build for API 19+ AFAIK, should I not be?